docs/developer-guide/sinks-guide.rst

```diff
@@ -0,0 +1,122 @@
+How to implement a custom Robusta sink
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+What Are Robusta Sinks?
+--------------
+
+`Sinks <https://docs.robusta.dev/master/catalog/sinks/index.htm>`_
```

Comment:
The link on this is broken. Please replace this and all the other links in this doc with Sphinx references.

https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html